### ISSUE TITLE
More events frequency options

### ISSUE THREAD

**Issue by FulcrumA** [AnarkisGaming/ATC-Feedback#139]
**Posting here an entry from Discord, as requested:**

> Rather than just frequency of events in general, a game start option allowing one to adjust rough categories of events (trader visits, recruit visits, wild beasts/infected/raiders attacks, special weather effects) would really help tailor one's game.

**Comment by IvanBajlo** [comment]
story mode has this option disabled probably to make balancing easier but individual scenario have this option open

![image](https://user-images.githubusercontent.com/91120787/192252944-332e319f-c7a1-466f-a5b0-dd0143361de0.png)

**Comment by FulcrumA** [comment]
Yes, though as mentioned, that one is for frequency of events in general, rather than allowing adjustment of different types of events independently.

**Comment by SerialKicked** [comment]
Sure, this was requested in the forums already. 
I'll add additional settings so you can customize individual event spawn rate based on their sub-type.
